Makefile

```diff
@@ -102,6 +102,7 @@ tests: $(SCOPE_BACKEND_BUILD_UPTODATE)
 
 lint: $(SCOPE_BACKEND_BUILD_UPTODATE)
 	./tools/lint -ignorespelling "agre " -ignorespelling "AGRE " .
+	./tools/shell-lint tools
```

Comment:
> why not run it on the whole repo?

To limit the amount of work I'm doing at once. Goal of this PR isn't to verify the shell script in scope, but rather to prove that the shell changes in weaveworks/build-tools#22 actually work.

Fixing scope's shell script is a separate PR.

> and why not make it part of lint?

Once we've fixed shell scripts in all of our repos. Having it here lets us ratchet it up incrementally without having to do one big 'fix everything' push.